src/Components/KYRContent/KYRContent.tsx

```diff
@@ -219,10 +219,10 @@ export const UniversalProtections: React.FC<KYRContentBoxProps> = ({
 );
 
 export const GoodCauseProtections: React.FC<
-  KYRContentBoxProps & { rent?: number }
+  KYRContentBoxProps & { rent?: number; subtitle?: string }
 > = ({
   title = undefined,
-  subtitle = "Protections you have under Good Cause",
```

Comment:
I don't think the changes to this file change the behavior here - since this line gives the default value for `subtitle` but if it's provided as a prop that overrides this value, and the `subtitle` prop is already included, inherited by `KYRContentBoxProps` from `ContentBoxProps`. We are already using this to give a different subtitle on results pages.

Reply:
oof, yeah good catch. I didn't even see that this same pattern is being used for different modules above